guide/12-enrich-data-with-thematic-information/part7_discover_and_enrich_standard_geographies.ipynb

```diff
@@ -0,0 +1,1165 @@
+{
```

Comment:
<p>From here, a variety of techniques can be used. However, with so many income and net worth variables, covariance needs to be addressed before we perform subsequent modeling steps. Using GeoEnrichment dramatically streamlines getting to this point, as it provides extremely easy access to thousands of demographic variables for modeling and analysis directly in Python, making it easy to integrate with data engineering pipelines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>